components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useContext, useState } from "react";
+import { useLocation } from "react-router";
+import ConfirmationModal from "../components/ConfirmationModal";
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { getGitpodService, gitpodHostUrl } from "../service/service";
+import { UserContext } from "../user-context";
+import { getCurrentTeam, TeamsContext } from "./teams-context";
+
+export default function TeamSettings() {
+    const [modal, setModal] = useState(false);
+    const [teamSlug, setTeamSlug] = useState('');
+    const { teams } = useContext(TeamsContext);
+    const { user } = useContext(UserContext);
+    const location = useLocation();
+    const team = getCurrentTeam(location, teams);
+
+    const close = () => setModal(false);
+
+    const deleteTeam = async () => {
+        if (!team || !user) {
+            return
+        }
+        await getGitpodService().server.deleteTeam(team.id, user.id);
+        document.location.href = gitpodHostUrl.asSettings().toString();
+    };
+
+    const settingsMenu = [
+        {
+            title: 'General',
+            link: [`/t/${team?.slug}/settings`]
+        }
+    ]
+
+    return <>
+        <PageWithSubMenu subMenu={settingsMenu} title='General' subtitle='Manage general team settings.'>
+            <h3>Delete Team</h3>
+            <p className="text-base text-gray-500 pb-4">Deleting this team will also remove all associated data with this team, including projects and workspaces. Deleted teams cannot be restored!</p>
+            <button className="danger secondary" onClick={() => setModal(true)}>Delete Account</button>
```

Comment:
**issue**: This returns an error in console and so feedback to the user when they are have member permissions, not owner permission, which also lacks visual feedback for the user. Hiding the project settings page for members (non-owners) could resolve this issue. WDYT? ❓